docs/pages/web/spinner.js

```diff
@@ -56,35 +45,25 @@ function SpinnerExample() {
           />
         </MainSection.Subsection>
       </MainSection>
-      <AccessibilitySection name={generatedDocGen?.displayName} />
+
+      <AccessibilitySection
+        name={generatedDocGen?.displayName}
+        description={`
+      Be sure to include \`accessibilityLabel\`. Labels should relate to the specific part of the product where Spinner is being used (e.g. "Loading homefeed" when used on the homefeed surface). Don't forget to localize the label!
+      `}
+      />
 
       <MainSection name="Variants">
-        <Example
+        <MainSection.Subsection
```

Comment:
Maybe we should add a variant "displayed" like the "paused" in Pulsar where we show the implementation.

Reply:
Yeah, this page needs a lot more love. I'm going to leave as-is for now (the goal was just to convert the examples to Sandpack), but we should revisit